hack/Dockerfile.windows

```diff
@@ -1,3 +1,3 @@
-FROM microsoft/nanoserver
+FROM mcr.microsoft.com/windows/nanoserver:1909
```

Comment:
AppVeyor can't 1909 -- if you add `APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019` to your `.appveyor.yml`, you can get 1809: https://www.appveyor.com/docs/windows-images-software/

(https://github.com/docker-library/golang/pull/317)

Reply:
you are better than Google. I won't report how many tabs I currently have open looking for guidance on this :)

Reply:
I can't take *full* credit; see https://github.com/docker-library/python/pull/446. :smile:

Reply:
In other words, you actually want `image: Visual Studio 2019` (no need for `APPVEYOR_BUILD_WORKER_IMAGE` since you're not a matrix).